<launch>
    <arg name="robot_name"/>
    <arg name="init_pose"/>



    <node name="spawn_minibot_model" pkg="gazebo_ros" type="spawn_model"
     args="$(arg init_pose) -urdf -param /robot_description -model $(arg robot_name)"
     respawn="false" output="screen" />

     <node name="$(arg robot_name)_revolve" type="lineF.py" pkg = "guard" >
         <param name="robot_name" type = "str" value = "$(arg robot_name)" /> 
         </node>


  <!-- <node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf" args="-urdf -model turtlebot3_$(arg model) -x $(arg x_pos) -y $(arg y_pos) -z $(arg z_pos) -param robot_description" /> -->


    <!-- <node pkg="robot_state_publisher" type="state_publisher" 
          name="robot_state_publisher" output="screen"/> -->

    <!-- The odometry estimator, throttling, fake laser etc. go here -->
    <!-- All the stuff as from usual robot launch file -->
</launch>